GOVERNANCE.md

```diff
@@ -0,0 +1,5 @@
+# Node.js GitHub Bot Project Governance
+
+The Node.js GitHub Bot is overseen by the Node.js Technical Steering Committee.
+https://github.com/nodejs/TSC
+
```

Comment:
> But for now, it is delegated to the members of the @nodejs/github-bot team.

To me that sounds like we should replace TSC w/github-bot team here.

Tbh as long as the bot stays in the nodejs org I'm happy.